fastify.js

```diff
@@ -262,7 +262,10 @@ function fastify (options) {
   const avvio = Avvio(fastify, {
     autostart: false,
     timeout: Number(options.pluginTimeout) || defaultInitOptions.pluginTimeout,
-    expose: { use: 'register' }
+    expose: {
+      ready: 'boot',
```

Comment:
Boot would need to be documented. I would prefer if it was overridden and not exposed as a new method.

Reply:
Update.
Not sure it is the nicest way, maybe assign `boot` to a symbol would it better?